### PR TITLE
Edit profile internationalization

### PR DESCRIPTION
**Pull request by dalink**
Internationalization for **_dashboard_about_me.html.slim**.
With translation for pt, en and fr language.
More work on internationalization with a specific focus on french language will come later.

### CODE REVIEW

File: app/views/catarse_bootstrap/users/_dashboard_about_me.html.slim [hunk 1]
```diff
@@ -79,6 +79,6 @@
         .w-col.w-col-4
         .w-col.w-col-4
           =  hidden_field_tag 'anchor', 'about_me'
-          = form.button :submit, 'Salvar',  class:'btn btn-large'
+          = form.button :submit, =t('users.edit.submit'),  class:'btn btn-large'
```
Comment (adrianob) [inline app/views/catarse_bootstrap/users/_dashboard_about_me.html.slim]: Syntax error here, the = is not needed

File: app/views/catarse_bootstrap/users/_dashboard_about_me.html.slim [hunk 2]
```diff
@@ -7,68 +7,68 @@
...
                 .w-col.w-col-4.w-sub-col
-                  = form.input :cover_image, label: '', as: :file, required: false, hint: (@user.cover_image.present? ? image_tag(@user.cover_image.base.url, size: '220x172#') : '' ) 
+                  = form.input :cover_image, label: '', as: :file, required: false, hint: (@user.cover_image.present? ? image_tag(@user.cover_image.base.url, size: '220x172#') : '' )
 
           - if current_user && current_user.admin?
             .w-row.u-marginbottom-30.card.card-terciary
               .w-col.w-col-5.w-sub-col
-                label.field-label.fontweight-semibold Endereço do seu perfil
-                label.field-label.fontsize-smallest.fontcolor-secondary Seu perfil público pode ter uma URL personalizada. Escolha uma fácil de guardar!    
+                label.field-label.fontweight-semibold =t('users.edit.profil_address_title')
```
Comment (adrianob) [inline app/views/catarse_bootstrap/users/_dashboard_about_me.html.slim]: Typo in 'profil'

**Comment by adrianob** [comment]
Thanks for the PR :)
We just need to move the translations to users.dashboard_about_me, so that we can use relative paths.

File: config/locales/catarse_bootstrap/views/users/edit.en.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+en:
+  users:
+    edit:
+      add_link: "+ Ajoutez a link"
```
Comment (adrianob) [inline config/locales/catarse_bootstrap/views/users/edit.en.yml]: Wrong language :p

**Comment by dalink** [comment]
Ok
I'll update with your comments.
I'll also add the full translation for the user edit screen

**Pull request closed without merging**